### PR TITLE
AWS amplify apps doc added

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — COMMENTED** [review]
Hi @JV0812 , it looks like their most recent logo is orange now (example [here](https://twitter.com/AWSAmplify)). Were the devs going to use this older red one in our UI?

**Comment by JV0812** [comment]
@kimsauce this logo is taken from the [AWS](https://aws.amazon.com/architecture/icons/) itself. And we are using the same icon in our App catalog.

File: docs/integrations/amazon-aws/aws-amplify.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* [Access Logs](https://docs.aws.amazon.com/amplify/latest/userguide/access-logs.html#using-access-logs)
+* [CloudTrail Logs](https://docs.aws.amazon.com/amplify/latest/userguide/logging-using-cloudtrail.html)
+
+## Setup
+
+This section has instructions for collecting logs and metrics for the Sumo Logic AWS Amplify integration.
+
+### Configure metrics collection
+
+* Collect CloudWatch Metrics with namespace AWS/AmplifyHosting using [AWS Kinesis Firehose for Metrics](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AmplifyHosting metrics and dimensions refer [AWS Amplify CloudWatch metrics](https://docs.aws.amazon.com/amplify/latest/userguide/access-logs.html#monitoring-with-cloudwatch).
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/aws-amplify.md:25]:
```suggestion
* Collect CloudWatch Metrics with namespace AWS/AmplifyHosting using [AWS Kinesis Firehose for Metrics](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AmplifyHosting metrics and dimensions refer to [AWS Amplify CloudWatch metrics](https://docs.aws.amazon.com/amplify/latest/userguide/access-logs.html#monitoring-with-cloudwatch).
```

File: docs/integrations/amazon-aws/aws-amplify.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Configure logs collection
+
+* Collect **Amazon CloudWatch Logs** for SSR apps using [AWS Kinesis Firehose for Logs](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/) source.
+
+* AWS Amplify stores access logs for all the apps you host in Amplify. Collect **Access Logs** using [Amazon S3](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-s3-source/) source.
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/aws-amplify.md:31]:
```suggestion
* AWS Amplify stores access logs for all the apps you host in Amplify. Collect **Access Logs** using [Amazon S3](/docs/send-data/hosted-collectors/amazon-aws/aws-s3-source/) source.
```

File: docs/integrations/amazon-aws/aws-amplify.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+* AWS Amplify is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or an AWS service in Amplify. CloudTrail captures all API calls for Amplify as events. The calls captured include calls from the Amplify console and code calls to the Amplify API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for Amplify. Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/amplify/latest/userguide/logging-using-cloudtrail.html) using [AWS CloudTrail](https://help.sumologic.com/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source.
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/aws-amplify.md:33]:
```suggestion
* AWS Amplify is integrated with AWS CloudTrail, a service that provides a record of actions taken by a user, role, or an AWS service in Amplify. CloudTrail captures all API calls for Amplify as events. The calls captured include calls from the Amplify console and code calls to the Amplify API operations. If you create a trail, you can enable continuous delivery of CloudTrail events to an Amazon S3 bucket, including events for Amplify. Collect [AWS CloudTrail Logs](https://docs.aws.amazon.com/amplify/latest/userguide/logging-using-cloudtrail.html) using [AWS CloudTrail](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source/) source.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Minor updates.

**Comment by JV0812** [comment]
Thanks for your review John. I have incorpotated your comments.

**Review by jpipkin1 — APPROVED** [review]

File: docs/integrations/amazon-aws/aws-amplify.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+id: aws-amplify
+title: AWS Amplify
+description: Learn about the collection process for the AWS Amplify service.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-amplify-logo.png')} alt="Thumbnail icon" width="50"/>
+
+AWS Amplify is a set of purpose-built tools and features that enables frontend web and mobile developers to quickly and easily build full-stack applications on AWS. For further details refer to [AWS documentation](https://docs.aws.amazon.com/amplify/latest/userguide/welcome.html).
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/aws-amplify.md:10]:
```suggestion
AWS Amplify is a set of purpose-built tools and features that enables frontend web and mobile developers to quickly and easily build full-stack applications on AWS. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/amplify/latest/userguide/welcome.html).
```

File: docs/integrations/amazon-aws/aws-amplify.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+### Configure metrics collection
+
+* Collect CloudWatch Metrics with namespace AWS/AmplifyHosting using [AWS Kinesis Firehose for Metrics](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For AWS/AmplifyHosting metrics and dimensions refer to [AWS Amplify CloudWatch metrics](https://docs.aws.amazon.com/amplify/latest/userguide/access-logs.html#monitoring-with-cloudwatch).
+
+### Configure logs collection
+
+* Collect **Amazon CloudWatch Logs** for SSR apps using [AWS Kinesis Firehose for Logs](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/) source.
+
+* AWS Amplify stores access logs for all the apps you host in Amplify. Collect **Access Logs** using [Amazon S3](/docs/send-data/hosted-collectors/amazon-aws/aws-s3-source/) source.
+
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/aws-amplify.md:32]:
```suggestion
* AWS Amplify stores access logs for all the apps you host in Amplify. Collect **Access Logs** using [Amazon S3](/docs/send-data/hosted-collectors/amazon-aws/aws-s3-source/) source.
```

**Pull request merged**